timebinmgr.c

```diff
@@ -29,6 +29,16 @@ setup_sync_points(void)
     SyncPoints[0].write_snapshot = 0; /* by default no output here. */
     SyncPoints[1].loga = log(All.TimeMax);
     SyncPoints[1].write_snapshot = 1;
+    /* Set the first SyncPoint to the last scheduled output.
+     * This ensures that the timeline does not change after
+     * save/load from an unscheduled output.*/
```

Comment:
The insertion sort should have taken care of this already..

Reply:
See the loop a few lines below.

Reply:
No, the insertion sort puts something like:
TimeInit, Out[2], Out[3]...
whereas this makes;
Out[1], Out[2], Out[3].
The point is to make the first SyncPoint the last *scheduled* output, which may not be the same as TimeInit.

Reply:
If TimeMin is used instead of TimeInit, then this loop is not needed. The code below will reconstruct the same set of planned sync points (which goverens ti and loga conversion) if OutputList and TimeMax are preserved as well.

Reply:
Yes, this is true. That is likely a simpler scheme, and it works also if the unplanned sync is before the first planned sync.